### DISCUSSION TITLE
[cli][error] Unable to parse JSON: Expecting value: line 1 column 1 (char 0) ('<!DOCTYPE html>\n<html lang="en" s ...)

### DISCUSSION THREAD

**Discussion by serseri0690** [streamlink/streamlink#5863] [General]
### Description

Hello. As of the beginning of the year, the current bigo downloader gives errors like this. Even if I tried the same program on different devices in different homes, it still gave negative results. How can I solve this?
![2024-02-24 01 30 30](https://github.com/streamlink/streamlink/assets/161086294/76d1d248-9f09-4e29-9106-3479bf65bac4)
![Ekran görüntüsü 2024-02-24 013018](https://github.com/streamlink/streamlink/assets/161086294/11bc662b-7526-443a-8c3b-b9eb4cf90a03)
>\Desktop\B1go>streamlink --http-no-ssl-verify -O bigo.tv/nurseem52 live --retry-streams 60   | ffmpeg -i pipe:0 -c:v copy -c:a copy -f mp4 "Videos\Aksin_idnurseem52_2024-02-24- 11508.mp4"
ffmpeg version 2024-02-22-git-76b2bb96b4-full_build-www.gyan.dev Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 12.2.0 (Rev10, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --pkg-config=pkgconf --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libaribcaption --enable-libdav1d --enable-libdavs2 --enable-libuavs3d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-dxva2 --enable-d3d11va --enable-libvpl --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libcodec2 --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      58. 39.100 / 58. 39.100
  libavcodec     60. 40.100 / 60. 40.100
  libavformat    60. 21.101 / 60. 21.101
  libavdevice    60.  4.100 / 60.  4.100
  libavfilter     9. 17.100 /  9. 17.100
  libswscale      7.  6.100 /  7.  6.100
  libswresample   4. 13.100 /  4. 13.100
  libpostproc    57.  4.100 / 57.  4.100
[cli][info] Found matching plugin bigo for URL bigo.tv/nurseem52
[cli][error] Unable to parse JSON: Expecting value: line 1 column 1 (char 0) ('<!DOCTYPE html>\n<html lang="en" s ...)
[cli][info] Waiting for streams, retrying every 60.0 second(s)

**Comment by bastimeyer** [comment]
Don't abuse the discussion forum for plugin issues. The rules are stated very clearly here.

In #5855 you failed to provide a full debug log. What you have posted is useless information because we don't even know which version of Streamlink you're running and whether you have made any other modifications. A full debug log (`--loglevel=debug`) without doing any other stuff like piping into ffmpeg would show this.

I have also clearly demonstrated that the plugin is working fine using the latest version of Streamlink.

And as you can see in the [plugin code](https://github.com/streamlink/streamlink/blob/6.6.2/src/streamlink/plugins/bigo.py#L29), all the plugin does is query a single API endpoint with the expectation of a JSON response. The error you're receiving is caused by a HTML response, which indicates that you're either blocked and they simply respond with different kind of data, or you're running some different/modified version of Streamlink.

Closing and locking this thread... If you want actual help, then start reading the plugin issue requirements we've set up here and adhere to them.
